src/pie-chart/utils.ts

```diff
@@ -131,12 +131,7 @@ export const balanceLabelNodes = (
   while ((leftSide && i >= 0) || (!leftSide && i < nodes.length)) {
     const node = nodes[i];
 
-    // Currently using dataset attributes to determine the base position.
-    // This implementation can be changed back to using `getBBox` when we drop IE11 support.
-    // Unfortunately, there is no good alternative for `getBBox` that is supported by IE11.
-    // `getBoundingClientRect` works for width and height calculations in SVG, but the x/y positions are inaccurate.
-    const x = parseFloat(node.getAttribute('data-x') || '0');
-    const y = parseFloat(node.getAttribute('data-y') || '0');
+    const { x, y } = node.getBBox();
```

Comment:
Are there any performance implications of this change? `getBBox` requires layout calculations while reading an attribute is not?

Do we really want to change this or just keep as is and remove the IE comment?

Reply:
No really, compared `data-x/y` with `getBBox` performance on pie-chart permutation page, the `getBBox` is a little slower (3-5 ms).

But I would rather agree, I personally don't find it beneficial to change the implementation. Let's remove the IE comment and close this PR.